Automattic/wp-calypso
# Pattern Library: Ensure that form submission disabled

Fixes issue: Automattic/dotcom-forge#6584

We add an event handler to forms with patterns to ensure that forms cannot be submitted.

Testing Instructions
---

1. Navigate to https://wordpress.com/patterns/contact#pattern-13979
3. Click on the `Contact us` heading in the pattern
4. Click <kbd>Tab</kbd> on your keyboard
5. The first input field should receive focus. Enter some text
6. Click <kbd>Tab</kbd> 
7. The second input field should receive focus. Enter some text
8. Click <kbd>Enter</kbd> 
9. Confirm that nothing happens, or you are not able to submit said form
